pioneer/packages/joy-utils/src/transport/tokenomics.ts

```diff
@@ -44,8 +44,8 @@ export default class TokenomicsTransport extends BaseTransport {
   }
 
   async calculateCouncilRewards (numberOfCouncilMembers: number) {
```

Comment:
I think the idea here was make it sort of universal, ie.:
```
const councilTermDurationRatio = termDuration / (termDuration + announcingPeriod + votingPeriod + revealingPeriod);
const avgCouncilRewardPerBlock = councilTermDurationRatio * totalCouncilRewardsPerBlock;
const avgCouncilRewardPerWeek = avgCouncilRewardPerBlock * (7 * 24 * 60 * 60) / 6;
```
Pinging @bwhm to confirm

Reply:
Not sure I understand, but:

To be precise, the `councilMembers` only earn rewards during their `newTermDuration`, ie. when there are no election ongoing, but they still have the same "power" during `announcingPeriod+votingPeriod+revealingPeriod`.

So to get the average (weekly) spending on CM rewards:
```
const weeklyAvgCouncilReward = activeCouncil.length * amountPerPayout * (blocksPerWeek / payoutInterval) * newTermDuration / (newTermDuration+announcingPeriod+votingPeriod+revealingPeriod) 
```
-> weeklyAvgCouncilReward = 6*18500*(100800/3600*)144000/201600 = 2220000

Reply:
I think our calculations are pretty much the same then.
The point is that this way we can get rid of the entrie `while` loop in `calculateCouncilRewards` that requires setting some arbitrary period in order to run the calculation (which doesn't seem necessary) @DzhideX

Reply:
Ok. I am very much for this as that was my least favorite part of the code as well.